gen3users/user_security.py

```diff
@@ -107,58 +109,80 @@ def _check_yaml(
 
         logging.info(f"========= Scanning file for {phsid} in open projects =========")
         found = False
-        # Track down open_data_reader
+
+        # Track down open data policies (anonymous or all_users)
         policies = content[auth_meth]["policies"]
+        open_policies = []
+
         for i in range(len(policies)):
```

Comment:
I don't think it makes sense to loop through policies _inside_ the loop for phsids. I'd populate the policies as a separate step beforehand to reduce number of loops needed.